main.yml

```diff
@@ -0,0 +1,34 @@
+on:
```

Comment:
is this file needed? looks like a copy of .github/workflows/main.yml

Reply:
Yeah, that's an error. Thanks.